app/src/main/java/org/simple/clinic/user/UserSession.kt

```diff
@@ -60,10 +60,12 @@ class UserSession @Inject constructor(
     return ongoingLoginEntry()
         .map { LoginRequest(UserPayload(it.phoneNumber!!, it.pin!!, it.otp)) }
         .flatMap { loginApi.login(it) }
-        .doOnSuccess { storeUserAndAccessToken(it) }
         .flatMap {
-          facilitySync
-              .sync()
+          storeUserAndAccessToken(it)
+              .andThen(Single.just(it))
```

Comment:
Instead of using `andThen`, can we use `toSingleDefault(it)` instead?  This applies to the next operator in the chain as well.

Reply:
Yeah, `toSingleDefault` looks semantically more correct.